### PR TITLE
Rebuild/documentation/fix link in flavor description

### LINKED ISSUE #256: Fix link in flavor description of standard 7.1 flavor for flavor_base_deps/packages/*

### LINKED ISSUE #262: python minimal flavor docu missing
## Background
The python minimal flavor is missing in flavors/README.md

## Solution

Add a description for the python3.6 minimal flavor to the READM.md.

### PR DESCRIPTION
**Pull request by tomuben**
#256:Fixed python links in flavor descriptions of standard 7.1 flavor
#262:Added description for existing flavor python-3.6-minimal-6.2.0
Links in standard-EXASOL-7.1.0/FLAVOR_DESCRIPTION.md were pointing to incorrect pip file.
Fixes #256.
Fixes #262.

### CODE REVIEW

**Comment by tkilias** [comment]
I recognized that the python minimal flavor is missing at https://github.com/exasol/script-languages-release/tree/rebuild/documentation/fix_link_in_flavor_description/flavors. Can you have look, if you can add it.

**Review by tkilias — CHANGES_REQUESTED** [review]
see previous comment

File: flavors/README.md [hunk 1]
```diff
@@ -35,6 +35,12 @@ We provide three flavors for different Exasol versions. The flavors only specify
   - Available languages: Java 11, Python 2.7, Python 3.6 and R 3.4
   - [Here](standard-EXASOL-6.2.0/FLAVOR_DESCRIPTION.md) you can find more details about the flavor, such as the provided packages.
 
+- python-3.6-minimal-6.2.0
```
Comment (tkilias) [inline flavors/README.md:38]: It gets its own section, as the r-4.0.minimal also got its own section. We have so to speak, one section per flavor family

File: flavors/README.md [hunk 2]
```diff
@@ -35,6 +35,12 @@ We provide three flavors for different Exasol versions. The flavors only specify
...
+  - Base Image: ubuntu:18.04
+  - Compatible Exasol Versions: 7.0.\*, 6.2.\*
+  - Available languages: Python 3.6
+  - [Here](https://github.com/exasol/script-languages/tree/master/flavors/python-3.6-minimal-EXASOL-6.2.0/FLAVOR_DESCRIPTION.md) you can find more details about the flavor, such as the provided packages.
```
Comment (tkilias) [inline flavors/README.md:42]: And a link in the [flavor description](https://github.com/exasol/script-languages/tree/master/flavors/python-3.6-minimal-EXASOL-6.2.0/FLAVOR_DESCRIPTION.md) of python-3.6 minimal is also broken. The link to 

https://github.com/exasol/script-languages/blob/master/flavors/python-3.6-minimal-EXASOL-6.2.0/flavor_base/flavor_base_deps/packages/pip3_packages

needs to be

https://github.com/exasol/script-languages/blob/master/flavors/python-3.6-minimal-EXASOL-6.2.0/flavor_base/flavor_base_deps/packages/python3_pip_packages

We really need the link checker :sweat_smile:

**Review by tkilias — CHANGES_REQUESTED** [review]

File: flavors/README.md (hunk 2 above)
Comment (tkilias) [reply flavors/README.md:44]: the submodule needs to be pushed and merged and then you need to update the submodule link to the new commit in master of script-languages

**Review by tkilias — CHANGES_REQUESTED** [review]

File: flavors/README.md (hunk 2 above)
Comment (tomuben) [reply flavors/README.md:44]: I created https://github.com/exasol/script-languages-release/issues/264

**Review by tkilias — APPROVED** [review]

**Pull request merged**